test/listen-dial.spec.js

```diff
@@ -192,6 +195,28 @@ describe('dial', () => {
     await listener.close()
   })
 
+  it('dial on path', async () => {
+    // Windows doesn't support unix paths
+    if (isWindows) return this.skip()
```

Comment:
you cannot use `this` with an arrow function above

Reply:
bah! fixed one but not the other. Fixed now.